common/txmgr/broadcaster.go

```diff
@@ -558,21 +558,34 @@ func (eb *Broadcaster[CHAIN_ID, HEAD, ADDR, TX_HASH, BLOCK_HASH, SEQ, FEE]) hand
 		eb.sequenceTracker.GenerateNextSequence(etx.FromAddress, *etx.Sequence)
 		return err, true
 	case client.Underpriced:
-		return eb.tryAgainBumpingGas(ctx, lgr, err, etx, attempt, initialBroadcastAt, retryCount+1)
+		bumpedAttempt, newErr := eb.replaceAttemptWithBumpedGas(ctx, lgr, err, etx, attempt)
+		if newErr != nil {
+			eb.SvcErrBuffer.Append(fmt.Errorf("%v, failed to create bumped attempt, %v", err, newErr))
+			return newErr, true
+		}
```

Comment:
`SvcErrBuffer is an ErrorBuffer that lets service owners track critical errors happening in the service.`. `Underpriced` error doesn't classify as such and can happen from time to time, so let's drop this.